novawallet/Modules/Staking/NominationPools/BondMore/Base/NominationPoolBondMoreBaseInteractor.swift

```diff
@@ -211,7 +211,11 @@ class NominationPoolBondMoreBaseInteractor: AnyProviderAutoCleaning, AnyCancella
 
         wrapper.addDependency(wrapper: assetInfoWrapper)
         assetExistenceCancellable = wrapper
```

Comment:
You forgot to create a single wrapper. Now, cancelling assetExistanceCancellable doesn't cancel assetInfoWrapper